test/functional/view.test.js

```diff
@@ -58,6 +58,7 @@ describe('Views', function() {
           ) {
             expect(r).to.exist;
             expect(err).to.not.exist;
+            delete commandResult.apiVersion;
```

Comment:
I wasn't sure about the best approach to take here. Below we currently use a deep equal check on the commandResult, which may or may not contain `apiVersion` depending on the presence of the MONGODB_API_VERSION environment variable.

We could use `containsSubset` instead of doing a deep equal check, but that allows the presence of extraneous keys we don't actually want to be there. Deleting the key seemed like a reasonable compromise, but I'm open to better ideas! Or perhaps just adding a comment would suffice.

Reply:
so there's a couple of different approaches possible here: 
1) if it is possible to know what the expected apiVersion is based on the test context, then we can optionally assert on it by pulling the expected object out into its own `const expectedResult = { create, viewOn, pipeline }` and then assigning the `apiVersion` with the expected value if we expect there to be one;
2) if it is possible to know whether or not the apiVersion is set at all, but not what it's set to, we could use the keys assertion (`expect(obj).to.have.all.keys([...])`) to check that all expected keys are present and then assert on the 3 that we know one at a time;
3) if it is not possible to know whether the apiVersion is even set, then just a comment to that end would be fine

Reply:
Went with option 1 👍